backend/src/main/java/com/ambacam/model/Requete.java

```diff
@@ -0,0 +1,147 @@
+package com.ambacam.model;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+@Entity
+public class Requete implements Serializable {
+	private static final long serialVersionUID = 3382958287248625084L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+
+	@NotEmpty(message = "The name must not be empty")
+	@NotNull(message = "The name must not be null")
+	@Column(columnDefinition = "varchar", nullable = false)
+	private String nom;
+
+	@ManyToOne
+	@JoinColumn(name = "status_requete_id", nullable = false, updatable = true)
+	private StatusRequete status;
+
+	private Date date;
+
+	@ManyToOne
+	@JoinColumn(name = "requete_id", nullable = false, updatable = false)
+	private Requerant requerant;
+
+	@ManyToOne
+	@JoinColumn(name = "operateur_id", nullable = false, updatable = true)
+	private Operateur operateur;
+
+	@ManyToOne

```

Comment:
is here manytomany or manytoone? if is manytoone please fix the property requetes in RequeteGroup

Reply:
That should be OneToMany, since it is a bidirectional relationship.
Thank you!